charts/fleet/charts/gitjob/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 name: gitjob
 description: Controller that run jobs based on git events
-version: v0.1.21
-appVersion: v0.1.21
+version: v0.1.22
```

Comment:
I think we have CRD changes in gitjob, can you also update that?

Reply:
fixed